### ISSUE TITLE
Miscellaneous documentation updates

### ISSUE THREAD

**Issue by rod-glover** [pacificclimate/climate-explorer-frontend#321]
1. From Trevor: I think the thing that’s important to note *somewhere* is that seasonal and annual RX1day and Rx5day are taken by taking maxes over the period of interest, as opposed to averaging the monthly maxes, which gives spurious results that can’t be interpreted easily. This can be a minor footnote somewhere but it’s not in any papers afaik.

1. Help > FAQ > #3 (polygons): EPSG:4325 -> EPSG:4326

**Comment by jameshiebert** [comment]
I would add two more points here:

3. Rename "Model" to "Model(s)" (since we have PCIC9/12 now)
4. Explain what PCIC9 and 12 are (ensemble mean of the models described [here](https://www.pacificclimate.org/data/statistically-downscaled-climate-scenarios).

**Comment by tqmurdock** [comment]
More on renaming. I would recommend adding "-mean" to PCIC9 and PCIC12 in those lists both so that it's clear and so that when we get around to adding 10/90 percentiles we can just add "PCIC12-90th%ile" etc. to the list below "PCIC12-mean" (which would remain the default)

**Comment by jameshiebert** [comment]
I agree with you @tqmurdock on the first part of this (adding "-mean" to the PCIC9/12 labels), but not the follow up process when more ensemble statistics are available.

There are more considerations to differentiating ensembles of model results from single model results that are not trivial. A special string in the name might be expedient for one or two special cases, but if we start adding a whole bunch of them (10th/25th/50th/75th/90th percentile, min/mean/max/stdev, all of which are required by the *current* P2A rules engine), any code that references models is going to devolve into unmaintainable. For example, the model context plot will cease to make any sense if it can't differentiate individual models from ensemble statistics.

**Comment by tqmurdock** [comment]
I appreciate the interest in avoiding a file of special references.

A couple of comments on this:

1. If P2A still needed things like min, max, stddev, there's no reason we would need to make those viewable (right now those are all internal and used only for the rules engine). I'm not sure it would be desirable to even make them visible.

2. I don't think that the impacts tab will require ANY of those special ensemble statistics. They were our poor person's way of getting at extremes and variability back in the day when we had monthly GCM results only. Now we have a pile of CLIMDEX indices based on daily results that do all that for us.

**Comment by tqmurdock** [comment]
I appreciate the interest in avoiding a file of special references.

A couple of comments on this:

1. If P2A still needed things like min, max, stddev, there's no reason we would need to make those viewable (right now those are all internal and used only for the rules engine). I'm not sure it would be desirable to even make them visible.

2. I don't think that the impacts tab will require ANY of those special ensemble statistics. They were our poor person's way of getting at extremes and variability back in the day when we had monthly GCM results only. Now we have a pile of CLIMDEX indices based on daily results that do all that for us.

**Comment by tqmurdock** [comment]
Further to this, here are possible options for where to draw the line before launching an updated P2A with CMIP5 projections. Any step beyond #1 that adds more than a week of (elapsed time) delay beyond what #1 would take is not worth it in my opinion.

1. Launch with CMIP5 RCP8.5 and impacts tab simply not present 
2. Same as above but with impacts tab taking you to the old (aka current) version of P2A
3. Same as #1 but with impacts tab still powered by CMIP3 even though everything is powered by CMIP5 RCP8.5 (would be fine for the user but I assume it would be a frankenstein to code)
